vectara_agentic/tools.py

```diff
@@ -143,6 +144,8 @@ def create_rag_tool(
             mmr_diversity_bias (float, optional): MMR diversity bias.
             include_citations (bool, optional): Whether to include citations in the response.
                 If True, uses markdown vectara citations that requires the Vectara scale plan.
+            fcs_threshold (float, optional): a threshold for factual consistency. 
+                Tool notifies caller it cannot respond if FCS it too high
```

Comment:
Wouldn't this notify if FCS is too low, not too high?

Reply:
yes, correct. Updated.